### ISSUE TITLE
Group members unable to edit document created in group

### ISSUE THREAD

**Issue by nicolaasmatthijs** [oaeproject/3akai-ux#4090]
```
How come that keeps happening? I created the document in the group area hoping that it would have group access as a default. Guess I'm doing something wrong. I just updated sharing permissions manually.
```

**Comment by ncaidin** [comment]
I seem to be able to reproduce.

1) Navigate to group
2) Go to library
3) Go to folder
4) Create document
5) Check permissions

The only way I see to be able to check permissions is by first adding someone, and then I can see existing permissions? I just tried testing with creating a document in the group, and creating a document by going to the group library. Then I choose to share, and add someone with whom to share, so I can see existing permissions. For those two cases, it does add the group permission but for the case of creating inside a document, it doesn't.

**Comment by nicolaasmatthijs** [comment]
Good catch. It looks like creating a document from a group correctly grants the group management permissions, but creating a document from a folder doesn't correctly grant the group management permissions.
